zfit_physics/models/pdf_cruijff.py

```diff
@@ -0,0 +1,100 @@
+from typing import Optional
+
+import zfit
+from zfit import z
+from zfit.util import ztyping
+from zfit.z import numpy as znp
+
+
+@z.function(wraps="tensor")
+def cruijff_pdf_func(x, mu, sigmal, alphal, sigmar, alphar):
+    """Caltulate the Cruijff PDF.
+
+    Args:
+        x: value(s) for which the PDF will be calculated.
+        mu: Mean value
+        sigmal: Left width parameter.
+        alphal: Left tail acceleration parameter.
+        sigmar: Right width parameter.
+        alphar: Right tail acceleration parameter.
+
+    Returns:
+        `tf.Tensor`: The calculated PDF values.
+
+    Notes:
+        Implementation from https://arxiv.org/abs/1005.4087
+    """
+    cond = znp.less(x, mu)
+
+    func = znp.where(
+        cond,
+        znp.exp(-0.5 * znp.square((x - mu) / sigmal) / (1 + alphal * znp.square((x - mu) / sigmal))),
```

Comment:
Factor out the znp.square? It should be ideally the same in compiled mode, but in eager mode, it's surely slower

Reply:
I'm not sure what that means. What equation form do you have in mind?

Reply:
I mean, doing something like... and as I wanted to give you an example, I just rewrote it quickly, it's more steps than I initially thought (even optimizes the traced mode too :D)